tests/log_fidelity/test_bgp_shutdown.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import pytest
+
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyzerError
+
+logger=logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+expected_log = "admin state is set to 'down'"
+defined_action = "sudo config bgp shutdown all"
```

Comment:
2 things here:
1. These are both constants so I would put them in `ALL_CAPS` to be consistent w/ the rest of the repo.
2. Since this file is testing BGP the names can be made more specific to the scenario (like "BGP_DOWN_EXPECTED_LOG_MESSAGE" for example)